roles/ceph-osd/tasks/pre_requisite.yml

```diff
@@ -58,3 +58,23 @@
   when:
     - cephx
     - item.copy_key|bool
+
+- set_fact:
+     osd_uuids: |
+      {%- set o=[] %}
+      {%- for device in  devices %}
+        {%- if o.append( ( inventory_hostname + device + ansible_date_time.iso8601_micro + "osd_uuids") | to_uuid ) %}
```

Comment:
Using `ansible_date_time.iso8601_micro ` will make the uuid change every time we run the playbook.
I think we should stick with `devices`, moreover we should start using devices like `/dev/disk/by-uuid/31ceb6f7-fc6f-4890-893d-561c233fcc1f` so we always have a unique identifier for a device.

This PR is addressing the device naming: #1128